runtime/src/main/java/org/corfudb/runtime/view/stream/StreamAddressSpace.java

```diff
@@ -162,7 +162,7 @@ private void removeAddresses(List<Long> addresses) {
     public void trim(long trimMark) {
         if (!Address.isAddress(trimMark)) {
             // If not valid address return and do not attempt to trim.
-            log.warn("trim: attempting to trim non-valid address {}", trimMark);
+            log.trace("trim: attempting to trim non-valid address {}", trimMark);
```

Comment:
should be good to wrapped with if log.isTraceEnabled?

Reply:
Done.

Reply:
Actually "parametrized messages" were added to slf4j not to pollute code with endless if(log.isTraceEnabled)
Please stop using if conditions for logs http://www.slf4j.org/faq.html#logging_performance